server/src/router/newsletter.js

```diff
@@ -1,10 +1,38 @@
-import express from 'express'
-const router=express.Router()
-import User from '../model/user.js'
-import {mailer} from '../controllers/newsletter.js'
-router.get('/',async (req,res)=>{
-    
-    res.send( await User.find({newsletterIsSubscribed:true}));
-})
-router.get('/email',mailer)
-export default router;
\ No newline at end of file
+import express from "express";
+const router = express.Router();
+import transporter from "../config/nodemailer.js";
+import User from "../model/user.js";
+import { mailer } from "../controllers/newsletter.js";
+
+router.get("/", async (req, res) => {
+  res.send(await User.find({ newsletterIsSubscribed: true }));
+});
```

Comment:
Don't you think this API is needed?

thoughts?? @Zolo-Ryan

Reply:
We didn't create that route. Just moved it down. It was from other prs